src/category_theory/shift.lean

```diff
@@ -220,6 +220,19 @@ section add_group
 variables (C) {A} [add_group A] [has_shift C A]
 variables (X Y : C) (f : X ⟶ Y)
 
```

Comment:
```suggestion
/-- Shifting by `i` is an equivalence. -/
```
Not necessarily the most helpful docstring, but it would be nice not to lose it! If you want to improve it instead please feel free :)